src/styles/global.css

```diff
@@ -6,8 +6,8 @@
   --blue-color: #caeaf8;
   --purple-color: #e4d0f2;
   --green-color: #cef8ca;
-  --grey-color: #8b8b8b;
-  --light-grey-color: #ececec;
+  --grey-color: #757575;
+  --light-grey-color: #eaeaea;
 
```

Comment:
@pavsidhu think you made this change, just wanna make sure this change doesn't get lost if we go back on it